.golangci.yml

```diff
@@ -44,43 +44,8 @@ linters:
   disable-all: true
   enable:
     #    - bodyclose
-    - deadcode
```

Comment:
Good point. I thought the same and because I was lazy I thought that this is the easiest way to 'mark' those just removed for now and bring the whole file back from git when time is right. All of this because we have had out-commented already some linters, I would have to add some markings that are commented out now just for go 1.18. I dunno what's the easiest, as said though that this is: *just bring the whole file back?*

Reply:
maybe I'll push a tag to mark the change which would make it easier to find the correct history.